content/geoip2/web-services/using-the-javascript-client.mdx

```diff
@@ -0,0 +1,241 @@
+---
+draft: false
+title: Using the JavaScript Client
+---
+
+Our GeoIP2 JavaScript client allows you to use GeoIP2 Precision services
+client-side without doing any server-side integration on your end.  While it is
+convenient, there are some caveats with its usage as some browser settings and
+add-ons (such as ad blockers) may prevent the GeoIP2 JavaScript API from
+successfully calling the web services, as well as unexpected usage spikes that
+you may want to monitor.
```

Comment:
This is a pretty long sentence. I think it can be broken up for clarity. Particularly it's not 100% clear how "as well as unexpected usage spikes that you may want to monitor." fits in. I think maybe it means:

> While it is convenient, there are some caveats with its usage. Some browser settings and
add-ons (such as ad blockers) may prevent the GeoIP2 JavaScript API from
successfully calling the web services. Additionally, there may be unexpected usage spikes which
you may want to monitor.

Probably also worth noting why usage might spike.